.readthedocs.yml

```diff
@@ -0,0 +1,23 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  builder: html
```

Comment:
I assume this means that the documentation is now not actually built through tox anymore?
If yes, we could adjust tox to use the same (or a very closely matching) command.

Reply:
Or I guess that would not really change much, so I'm fine with leaving it the way it is.

Reply:
Judging from other repos having the same setup there is no way to have read the docs actually run the tox command. Instead it just directly calls sphinx-build.